component/models/signature/signer/threshold_bbs_signer.go

```diff
@@ -0,0 +1,155 @@
+package signer
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/hyperledger/aries-framework-go/component/kmscrypto/crypto/primitive/bbsplusthresholdpub"
+)
+
+// baseSignatureSigner defines a base signature signer.
+type baseSignatureSigner struct {
+	keyType   string
+	curve     string
+	algorithm string
+}
+
+func (sv baseSignatureSigner) KeyType() string {
+	return sv.keyType
+}
+
+func (sv baseSignatureSigner) Curve() string {
+	return sv.curve
+}
+
+func (sv baseSignatureSigner) Algorithm() string {
+	return sv.algorithm
+}
+
+// ThresholdBBSG2SignaturePartySigner defines a party signer based on the baseSignatureSigner.
+// Party signer signs a credential and produces a partial signature based on its pregenerated presignature.
+type ThresholdBBSG2SignaturePartySigner struct {
+	partyPrivKeyBytes []byte
+	indices           [][]int
```

Comment:
Please add a comment here explaining this 2d array.

Reply:
The commit 14e2dcdffca12f285d1fd857938da8ebb9c6888a solved this issue